.github/workflows/ci-nightly.yml

```diff
@@ -35,8 +35,8 @@ jobs:
       run: lerna run prettier:diff
 
     - name: Testing unit tests
-      run: lerna run test --stream
-
+      run: lerna run test --stream   
+      
```

Comment:
Should these added spaces be removed?

Reply:
@chrisdholt this space should have been there before. It creates spacing between tasks which is consistent across all yaml pipelines.  So this is accurate.

Reply:
@awentzel I think an empty return is expected but I'm not sure about the (several) empty spaces after the text and on the next line. I won't block on it but the above is different than an empty return. Not sure if it'll cause issues or not though.

Reply:
This won't cause any issues.